### DISCUSSION TITLE
Is panopticon a person?

### DISCUSSION THREAD

**Discussion by brianteeman** [akeeba/panopticon#685] [General]
This may be a "lost in translation" moment but whenever I see strings such as "Where did I get this data from?" it grates on me. It is unusual for software to "speak" to me. I expect the message to be more neutral such as "Where does this data come from?"

But maybe it is just me and it is normal for everyone else.

**Comment by nikosdion** [comment]
Item one: Generally speaking, people ascribe agency to inanimate objects. Most users ask "why did X do Y" instead of "why did Y happen with X".

Item two: It is very important that users understand the provenance of data for these two displays.

Therefore: Having the software "talk" to users in a way that implies agency is far more likely to get their attention than using an impersonal writing style.

You will also notice that this kind of language is only used in the aforementioned message and certain error messages, i.e. when I want the user to pay attention and understand that the software does not, in fact, have any kind of agency. You see what I did there?

Consider this an experiment in hacking users' attention. So far, it seems to be working. People don't ask about anything that uses personification in its language. They only ask about things which use a more neutral language.

**Reply by brianteeman** [reply]
as i said - its lost in translation as it is very very strange for me to see this.

**Reply by nikosdion** [reply]
Note that it's been tried before, with success https://dl.acm.org/doi/10.1145/2016911.2016934 I didn't invent this.

Still, you are telling me that your tone of voice preferences are more important than my time which is in too short supply.

No worries, I can make us both happy. I will change the language string, but you are now responsible for answering all issues and discussions which touch this subject, and/or the change of the language string.
